include/downloader.h

```diff
@@ -193,10 +193,11 @@ class Downloader
    * User should call `update` on the returned `Download` to have an accurate status.
    *
    * @param uri: The uri of the thing to download.
+   * @param downloadDir: The download directory where the thing should be stored.
```

Comment:
Add a note that it takes precedence over the download directory passed via `options` (if any)